.vscode/settings.json

```diff
@@ -26,5 +26,8 @@
   },
   "[ruby]": {
     "editor.tabSize": 2
+  },
+  "files.exclude": {
+    "bazel-*": true
```

Comment:
cc @jvilk-stripe for `.vscode` change. @azdavis maybe if you could mention what this achieves

Reply:
if you open vscode on the sorbet repo (i.e. `cd sorbet; code .`) _every_ file shows up in the file explorer, even generated files like those in bazel-build, etc. This makes it so that those don't appear in the sidebar anymore (and also excludes them from things like search results, which was my ultimate goal here).

Reply:
This change is fine with me.